subiquity/common/apidef.py

```diff
@@ -306,16 +306,10 @@ class reformat_disk:
                 def POST(data: Payload[ReformatDisk]) \
                     -> StorageResponseV2: ...
 
-            class potential_boot_disks:
```

Comment:
I assume the desktop installer is the only consumer we know of for this endpoint. When we backport this fix, should we still keep the existing endpoint? This feels like a breaking change.